ArduCopter/mode_flip.cpp

```diff
@@ -126,17 +131,24 @@ void ModeFlip::run()
         if (flip_angle >= 4500) {
             if (roll_dir != 0) {
                 // we are rolling
-            _state = FlipState::Roll;
+                _state = FlipState::Roll;
             } else {
                 // we are pitching
                 _state = FlipState::Pitch_A;
-        }
+            }
         }
         break;
 
     case FlipState::Roll:
-        // between 45deg ~ -90deg request 400deg/sec roll
-        attitude_control->input_rate_bf_roll_pitch_yaw(FLIP_ROTATION_RATE * roll_dir, 0.0, 0.0);
+        // keep target aircraft from getting too far away from actual aircraft
```

Comment:
Won't the attitude controller do this stuff for us based on accel limits?

Reply:
It seems not. 

However, if we do use the knock down won't we end up timing out?

Reply:
Yeah. I still have to address that. I’ll come up with with something